scripts/ranks.ts

```diff
@@ -264,8 +264,8 @@ async function main() {
       const bpe = await downloadBpe(data);
 
       if (lib === "js") {
-        bpe.pat_str = bpe.pat_str.replace(
-          /\(\?i:(.*?)\)/,
+        bpe.pat_str = bpe.pat_str.replaceAll(
+          /\(\?i:(.*?)\)/g,
```

Comment:
It's necessary to do this as a global replace because `?i:` is now present several times in pat_str